query/querytest/compile.go

```diff
@@ -34,43 +30,9 @@ var opts = append(
 )
 
 func BucketAwareQueryTestHelper(t *testing.T, tc BucketAwareQueryTestCase) {
+	t.Skip("BucketsAccessed needs re-implementing; see https://github.com/influxdata/influxdb/issues/13278")
 	t.Helper()
-
-	now := time.Now().UTC()
-	got, err := flux.Compile(context.Background(), tc.Raw, now)
-	if (err != nil) != tc.WantErr {
-		t.Errorf("error compiling spec error: %v, wantErr %v", err, tc.WantErr)
-		return
-	}
-	if tc.WantErr {
-		return
-	}
-	if tc.Want != nil {
-		tc.Want.Now = now
-		if !cmp.Equal(tc.Want, got, opts...) {
-			t.Errorf("unexpected specs -want/+got %s", cmp.Diff(tc.Want, got, opts...))
-		}
-	}
-
-	var gotReadBuckets, gotWriteBuckets []platform.BucketFilter
-	if tc.WantReadBuckets != nil || tc.WantWriteBuckets != nil {
-		gotReadBuckets, gotWriteBuckets, err = query.BucketsAccessed(got, nil)
-		if err != nil {
-			t.Fatal(err)
-		}
-	}
-
-	if tc.WantReadBuckets != nil {
-		if diagnostic := verifyBuckets(*tc.WantReadBuckets, gotReadBuckets); diagnostic != "" {
-			t.Errorf("Could not verify read buckets: %v", diagnostic)
-		}
-	}
-
-	if tc.WantWriteBuckets != nil {
-		if diagnostic := verifyBuckets(*tc.WantWriteBuckets, gotWriteBuckets); diagnostic != "" {
-			t.Errorf("Could not verify write buckets: %v", diagnostic)
-		}
-	}
```

Comment:
isn't it better to comment out these lines to not lose track of them?

Reply:
Yes, I think you're right.